release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,1612 @@
+# System.Runtime.Intrinsics.X86
```

Comment:
System.Runtime.Intrinsics.X86 LGTM

This is the addition of the `[ConstantExpected]` attribute and the new AVX-512 surface area (`Avx512F`, `Avx512BW`, `Avx512CD`, `Avx512DQ`, `Avx512Vbmi`, etc).

CC. @dotnet/area-system-runtime-intrinsics